src/upload.ts

```diff
@@ -359,11 +359,42 @@ async function uploadVideo(videoJSON: Video, messageTransport: MessageTransport)
         await page.click(
             "ytcp-video-monetization-edit-dialog.cancel-button-hidden .ytcp-video-monetization-edit-dialog #save-button"
         );
+
+        await page.waitForTimeout(1500);
+
+        await page.waitForXPath(nextBtnXPath);
+        next = await page.$x(nextBtnXPath);
+        await next[0].click();
+
+        try {
+            await page.waitForSelector(
+                ".ytpp-self-certification-questionnaire .ytpp-self-certification-questionnaire #checkbox-container",
+                { visible: true, timeout: 10000 }
+            );
+            await page.evaluate(() =>
+                //@ts-ignore
+                document.querySelector(".ytpp-self-certification-questionnaire .ytpp-self-certification-questionnaire #checkbox-container").click()
+            );
+
+            await page.waitForTimeout(1500);
+
+            await page.waitForSelector(
+                ".ytpp-self-certification-questionnaire .ytpp-self-certification-questionnaire #submit-questionnaire-button",
+                { visible: true }
+            );
+            await page.evaluate(() =>
+                //@ts-ignore
+                document.querySelector(".ytpp-self-certification-questionnaire .ytpp-self-certification-questionnaire #submit-questionnaire-button").click()
+            );
+
+            await page.waitForXPath(nextBtnXPath);
+            next = await page.$x(nextBtnXPath);
+            await next[0].click();
+
+            await page.waitForTimeout(1500);
+        } catch {}
```

Comment:
Why are we hidding errors ?

Reply:
because I didn't have time for it at the moment, I will improve the entire section (for monetization) when it will work without problems and nothing new will happen (like new section, etc.)

Reply:
@pierreminiggio Changed here: https://github.com/fawazahmed0/youtube-uploader/pull/206